### PR TITLE
fix 0.0.0.0 -> backbone issue

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

File: examples/l3out_ospf_external_policy/main.tf [hunk 1]
```diff
@@ -19,9 +19,9 @@ resource "aci_l3out_ospf_external_policy" "example" {
   l3_outside_dn     = aci_l3_outside.example.id
   annotation        = "example"
   area_cost         = "1"
-  area_ctrl         = "redistribute,summary"
+  area_ctrl         = ["redistribute"]
   area_id           = "0.0.0.1"
-  area_type         = "nssa"
```
Comment (lhercot) [inline examples/l3out_ospf_external_policy/main.tf:24]: why this change?

File: examples/l3out_ospf_external_policy/main.tf [hunk 2]
```diff
@@ -19,9 +19,9 @@ resource "aci_l3out_ospf_external_policy" "example" {
   l3_outside_dn     = aci_l3_outside.example.id
   annotation        = "example"
   area_cost         = "1"
-  area_ctrl         = "redistribute,summary"
```
Comment (lhercot) [inline examples/l3out_ospf_external_policy/main.tf:22]: missing summary?

**Review by lhercot — CHANGES_REQUESTED** [review]

File: examples/l3out_ospf_external_policy/main.tf (hunk 1 above)
Comment (shrsr) [reply examples/l3out_ospf_external_policy/main.tf:24]: reverted it

File: examples/l3out_ospf_external_policy/main.tf (hunk 2 above)
Comment (shrsr) [reply examples/l3out_ospf_external_policy/main.tf:22]: area_ctrl is actually a list that takes only one argument

Comment (shrsr) [reply examples/l3out_ospf_external_policy/main.tf:22]: done

**Review by lhercot — APPROVED** [review]
LGTM
